mlir/include/mlir/Dialect/EmitC/IR/EmitC.h

```diff
@@ -43,6 +43,16 @@ bool isIntegerIndexOrOpaqueType(Type type);
 
 /// Determines whether \p type is a valid floating-point type in EmitC.
 bool isSupportedFloatType(mlir::Type type);
+
+/// Determines whether \p type is a emitc.size_t/ssize_t type.
+bool isSizeTType(mlir::Type type);
+
+/// Determines whether \p type is a emitc.ssize_t type.
+bool isSignedSizeTType(mlir::Type type);
```

Comment:
I don't see this function adding value, but it makes the reader wonder what it does. Can we remove it and just use `isa<emitc::SizeTType>` directly?